### PR TITLE
Add user list in chat + UI fix

### LINKED ISSUE #14: User can see the list of user in the channel
User wants to be able to know which users are present in a channel

### PR DESCRIPTION
**Pull request by adrienmo**
To close #14

### CODE REVIEW

File: api/lib/api/web/channels/chat_channel.ex [hunk 1]
```diff
@@ -21,8 +21,10 @@ defmodule Api.Web.ChatChannel do
       )
 
       paged_messages = Chats.list_messages(channel)
+      user_status = Chats.list_users(channel)
 
       response = %{
+        userStatus: user_status,
```
Comment (ghost) [inline api/lib/api/web/channels/chat_channel.ex]: Might be better to use a view to have the explicit list of what's required for this part of the response. My concern is if we change `Chats.list_users(channel)` we might make a mistake and expose data we shouldn't (password_hash for example)
Changing a little bit `list_users` you could use `Api.Web.UserView`

File: api/lib/api/chats/chats.ex [hunk 2]
```diff
@@ -185,6 +185,24 @@ defmodule Api.Chats do
...
+  Returns the list of users from a channel.
+
+  ## Examples
+
+      iex> list_users(channel)
+      [%User{}, ...]
+
+  """
+  def list_users(channel) do
+    User
```
Comment (ghost) [inline api/lib/api/chats/chats.ex]: Why not using `Repo.preload(channel, :users)`

**Review by ghost — CHANGES_REQUESTED** [review]

File: api/lib/api/chats/chats.ex (hunk 2 above)
Comment (adrienmo) [reply api/lib/api/chats/chats.ex]: Updated accordingly

File: api/lib/api/web/channels/chat_channel.ex (hunk 1 above)
Comment (adrienmo) [reply api/lib/api/web/channels/chat_channel.ex]: Updated accordingly

**Review by ghost — APPROVED** [review]

**Pull request merged**